aws-auth-cognito/src/main/java/com/amplifyframework/statemachine/codegen/data/AuthConfiguration.kt

```diff
@@ -55,7 +56,10 @@ internal data class AuthConfiguration internal constructor(
                     pluginJson.optJSONObject("Auth")
                         ?.optJSONObject(configName)
                         ?.optString("authenticationFlowType")
-                )
+                ),
+                pinpointAppId = pluginJson.optJSONObject("PinpointAnalytics")
```

Comment:
Looks like there is some disparity with iOS on how we are reading the AppId: https://github.com/aws-amplify/amplify-swift/pull/2371/files#diff-9cb850c74bd6d5694784f8790c80f355906e130d46e5af241f1d17bb3f576e45R27

I think what we have makes more sense. Please sync with iOS.

Reply:
Correct, we discovered this and are setting up a discussion. v1 read with iOS property as well but it doesn't appear that CLI event adds that field. There are a few bugs around this feature in v1 we will also be discussing. I'm moving this PR to draft until we resolve how to handle these.